website/docs/cli/commands/terraform/terraform-workspace.md

```diff
@@ -3,11 +3,12 @@ title: atmos terraform workspace
 sidebar_label: workspace
 sidebar_class_name: command
 id: workspace
-description: This command calculates the `terraform` workspace for an `atmos` component (from the context variables and stack config), then runs `terraform init -reconfigure`, then selects the workspace by executing the `terraform workspace select` command.
+description: This command calculates the `terraform` workspace for an Atmos component (from the context variables and stack config), then runs `terraform init -reconfigure`, then selects the workspace by executing the `terraform workspace select` command.
 ---
 
 :::note Purpose
-Use this command to calculate the `terraform` workspace for an `atmos` component (from the context variables and stack config), then run `terraform init -reconfigure`, then select the workspace by executing the `terraform workspace select` command.
+Use this command to calculate the `terraform` workspace for an Atmos component (from the context variables and stack config), then
```

Comment:
```suggestion
Use this command to calculate the `terraform` workspace for an Atmos component (from the context variables and stack config). It will
```